src/backend/distributed/sql/downgrades/citus--10.0-1--9.5-1.sql

```diff
@@ -9,13 +9,70 @@ DROP VIEW public.citus_tables;
+DROP FUNCTION pg_catalog.citus_update_shard_statistics(bigint);
+DROP FUNCTION pg_catalog.citus_update_table_statistics(regclass);
+DROP FUNCTION pg_catalog.citus_copy_shard_placement(bigint, text, integer, text, integer, bool, citus.shard_transfer_mode);
+DROP FUNCTION pg_catalog.citus_move_shard_placement(bigint, text, integer, text, integer, citus.shard_transfer_mode);
+
+ALTER FUNCTION citus_conninfo_cache_invalidate()
+RENAME TO master_conninfo_cache_invalidate;
+ALTER FUNCTION citus_dist_local_group_cache_invalidate()
+RENAME TO master_dist_local_group_cache_invalidate;
+ALTER FUNCTION citus_dist_node_cache_invalidate()
+RENAME TO master_dist_node_cache_invalidate;
+ALTER FUNCTION citus_dist_object_cache_invalidate()
+RENAME TO master_dist_object_cache_invalidate;
+ALTER FUNCTION citus_dist_partition_cache_invalidate()
+RENAME TO master_dist_partition_cache_invalidate;
+ALTER FUNCTION citus_dist_placement_cache_invalidate()
+RENAME TO master_dist_placement_cache_invalidate;
+ALTER FUNCTION citus_dist_shard_cache_invalidate()
+RENAME TO master_dist_shard_cache_invalidate;
+ALTER FUNCTION citus_drop_all_shards(regclass, text, text)
+RENAME TO master_drop_all_shards;
 
 DROP VIEW pg_catalog.time_partitions;
 DROP FUNCTION pg_catalog.time_partition_range(regclass);
 
 DROP FUNCTION pg_catalog.citus_set_coordinator_host(text,int,noderole,name);
 
+CREATE FUNCTION pg_catalog.master_modify_multiple_shards(text)
+    RETURNS integer
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$master_modify_multiple_shards$$;
+COMMENT ON FUNCTION master_modify_multiple_shards(text)
+    IS 'push delete and update queries to shards';
+
+CREATE FUNCTION pg_catalog.master_create_distributed_table(table_name regclass,
+                                                           distribution_column text,
+                                                           distribution_method citus.distribution_type)
+    RETURNS void
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$master_create_distributed_table$$;
+COMMENT ON FUNCTION pg_catalog.master_create_distributed_table(table_name regclass,
+                                                               distribution_column text,
+                                                               distribution_method citus.distribution_type)
+    IS 'define the table distribution functions';
+
+CREATE FUNCTION pg_catalog.master_create_worker_shards(table_name text, shard_count integer,
+                                                       replication_factor integer DEFAULT 2)
+    RETURNS void
+    AS 'MODULE_PATHNAME'
+    LANGUAGE C STRICT;
+
+#include "../udfs/citus_drop_trigger/9.5-1.sql"
 #include "../udfs/citus_total_relation_size/7.0-1.sql"
 #include "../udfs/upgrade_to_reference_table/8.0-1.sql"
 #include "../udfs/undistribute_table/9.5-1.sql"
 #include "../udfs/create_citus_local_table/9.5-1.sql"
+#include "../udfs/master_drain_node/9.2-1.sql"
```

Comment:
can you remove this line? as far as I know we did not drop master_drain_node, so no need to recreate it.